File: anagram-check/boilerplate-full/function.java

  import java.util.*;
  
  class Main {
      public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		String s1 = scanner.nextLine();
        String s2 = scanner.nextLine();
		Main m = new Main(); 
		Solution solution = m.new Solution();
		boolean result = solution.isAnagram(s1, s2);
		System.out.println(result);
		scanner.close();
      }

	  ##USER_CODE_HERE##
  }
      

File: anagram-check/boilerplate-full/function.js

	##USER_CODE_HERE##
		
  const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
  const s1 = (x) => x(input.shift());
const s2 = (x) => x(input.shift());
  const result = isAnagram(s1, s2);
  console.log(JSON.stringify(result));
      